README.md

```diff
@@ -11,16 +11,11 @@ A unified publishing application for content on GOV.UK
 
 ## Technical documentation
 
-This is a Ruby on Rails application. You can use the GOV.UK Docker environment to run the application and its tests with all the necessary dependencies; **see the [README](https://github.com/alphagov/govuk-docker#usage) for more details**.
```

Comment:
Whilst it's common knowledge to GOV.UK developers, I don't think it hurts to explicitly state that a given repo is a Ruby on Rails app. We do after all have some special cases like [router](https://github.com/alphagov/router) or [govuk-dependencies](https://github.com/alphagov/govuk-dependencies), which are not Rails apps, so somebody knew to GOV.UK might appreciate the distinction. For the sake of seven words, I'd probably just leave this in.

Reply:
I was going to disagree:

- This is actually (7 words) * (most of our repos), which is a classic instance of DRY.
- Crucially, if we're going to have boilerplate, it should be actionable boilerplate.

However, I thought of some documentation this could usefully link to, so I've amended the last commit with it.

Does that address your concern?

Reply:
Yes, that looks great. Thanks.

Reply:
Did we want to apply those changes to https://github.com/alphagov/email-alert-service/pull/437 and https://github.com/alphagov/email-alert-service/pull/437 too?

Reply:
Sure can, if you'd like to merge and deploy (otherwise Slack will start whining). The line only makes sense for Email Alert API, so I've just updated that (Email Alert Service already has blurb explaining how it works).